test/stable/common/hpa.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+require_relative '../../test_helper'
+
+class Test < ChartTest
```

Comment:
Similar to the other PR's: we should rename this file to hpa_spec.rb, otherwise it won't be picked up by the test framework. Also please add the whitespace between the different test cases here as well.

Reply:
Fixed and also fixed the other files while at it.